### PR TITLE
@ #19 | should add docs how to use docker client from the host to connect to docker daemon from the guest machine

### PR DESCRIPTION
**Pull request by hoatle**
connect #19

### CODE REVIEW

File: docs/docker-daemon-remote-access.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Docker Daemon Remote Access
+
+Similar to this https://kubernetes.io/docs/setup/minikube/#use-local-images-by-re-using-the-docker-daemon,
+this is useful in some some circumstances. You can follow this guide to enable remote access from
```
Comment (hoavt) [inline docs/docker-daemon-remote-access.md]: remove 1 "some"

File: docs/docker-daemon-remote-access.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+the host machine docker client to the VM docker daemon.
+
+
+## Add dockerd options
+
+You and add the `-H` dockerd options via ansible configuration on the `teracy-dev-entry/config_override.yaml`
```
Comment (hoavt) [inline docs/docker-daemon-remote-access.md]: --> You can add ....

Comment (hoavt) [reply docs/docker-daemon-remote-access.md]: in the `teracy-dev-entry/config_override.yaml`

File: docs/docker-daemon-remote-access.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+with the following config:
```
Comment (hoavt) [inline docs/docker-daemon-remote-access.md]: config --> configuration

File: docs/docker-daemon-remote-access.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```yaml
+teracy-dev-k8s:
+  ansible:
+    host_vars:
+      # workaround for TCP remote access to docker daemon from the host to the VM
+      docker_log_opts: "-H fd:// -H tcp://0.0.0.0:2375 --log-opt max-size=50m --log-opt max-file=5"
+```
+
+and then `$ vagrant reload --provision` to get that config take effect.
```
Comment (hoavt) [inline docs/docker-daemon-remote-access.md]: configuration to take effect.

**Review by hoavt — COMMENTED** [review]
Please check my comments @hoatle

**Comment by hoatle** [comment]
comments applied, thank you @hoavt

**Review by hieptranquoc — APPROVED** [review]

**Pull request merged**